R/apply_reporting_structure.R

```diff
@@ -18,15 +19,15 @@
 #' @examples
 #' # Generate retrospective triangles from truncated triangles
 #' trunc_rts <- truncate_triangles(example_reporting_triangle, n = 2)
-#' retro_rts <- construct_triangles(trunc_rts)
+#' retro_rts <- apply_reporting_structures(trunc_rts)
 #'
 #' # With custom structure
-#' retro_rts_custom <- construct_triangles(
+#' retro_rts_custom <- apply_reporting_structures(
 #'   trunc_rts,
 #'   structure = 2
 #' )
 #' retro_rts_custom
-construct_triangles <- function(truncated_reporting_triangles,
+apply_reporting_structures <- function(truncated_reporting_triangles,
                                 structure = 1,
```

Comment:
Maybe out of scope for this PR but should we consider calling this `reporting_structure` rather than `structure`? Only ask because `structure` does have another meaning in R and it would connect this function's name and its arguments.

Reply:
Closing this but please do open an issue for discussion